tests/test_controller.py

```diff
@@ -102,7 +102,7 @@ def test_control_dynamical(control_type):
 
     vals_start = Controller.get_sensors()
 
-    v_peep = random.randint(5, 10)
+    v_peep = 5
```

Comment:
this should change, and incorporate the results of our meeting tomorrow

Reply:
When therapists set a PEEP mechanical valve, they probably would like some feedback on the GUI on what the effective PEEP is. Perhaps the user does not set "desired PEEP" as a monitor, but rather sees the "average PEEP" over the last several cycles, so they can adjust slightly if desired.